package.json

```
@@ -3,22 +3,27 @@
    "version": "0.0.1",
    "description": "Awesome project developed with TypeORM.",
    "devDependencies": {
+      "@types/bcryptjs": "^2.4.2",
       "@types/node": "^8.0.29",
       "ts-node": "3.3.0",
       "typescript": "3.3.3333"
    },
    "dependencies": {
       "@types/express": "^4.17.13",
+      "@types/jsonwebtoken": "^8.5.8",
       "@types/uuid": "^8.3.4",
+      "bcryptjs": "^2.4.3",
       "express": "^4.17.3",
+      "jsonwebtoken": "^8.5.1",
       "pg": "^8.4.0",
       "reflect-metadata": "^0.1.10",
       "ts-node-dev": "^1.1.8",
       "typeorm": "0.2.41",
       "uuid": "^8.3.2"
```

Acho que é importante especificar versão afim de evitar breaking changes em updates posteriores.

```suggestion
      "bcryptjs": "2.4.3",
```